packages/netlify-cms-core/src/reducers/editorialWorkflow.ts

```diff
@@ -121,13 +140,48 @@ const unpublishedEntries = (state = Map(), action: EditorialWorkflowAction) => {
       );
 
     case UNPUBLISHED_ENTRY_PUBLISH_SUCCESS:
+      return state.withMutations(map => {
+        map.deleteIn(['entities', `${action.payload!.collection}.${action.payload!.slug}`]);
+        map.deleteIn([
```

Comment:
This line seems redundant as we're already deleting a parent path in the previous line.

Reply:
So what should I do when the case is UNPUBLISHED_ENTRY_PUBLISH_SUCCESS ?